spring-data-relational/src/test/java/org/springframework/data/relational/core/mapping/RelationalPersistentEntityImplUnitTests.java

```diff
@@ -32,90 +32,88 @@
  * @author Mark Paluch
  * @author Mikhail Polivakha
  */
-public class RelationalPersistentEntityImplUnitTests {
+class RelationalPersistentEntityImplUnitTests {
 
-	RelationalMappingContext mappingContext = new RelationalMappingContext();
+	private RelationalMappingContext mappingContext = new RelationalMappingContext();
```

Comment:
I thought we don't make stuff private in tests.

Reply:
We have no strong guidance about private/non-private. This is mostly an effect of the JUnit 5 migration aid that reduces visibility to the lowest possible visibility requirement.